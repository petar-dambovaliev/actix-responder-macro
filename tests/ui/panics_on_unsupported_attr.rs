extern crate actix_responder_macro;
extern crate typed_builder;

use actix_responder_macro::actix_responder;
use serde::{Deserialize, Serialize};
use typed_builder::TypedBuilder;

#[actix_responder(imaginary_attr = "builder(default)")]
#[derive(Serialize, Deserialize, Debug, TypedBuilder, Default)]
struct SuccessResp {
    success: bool,
}

fn main() {}
