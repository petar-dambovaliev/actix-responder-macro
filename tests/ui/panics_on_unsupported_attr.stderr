error: Invalid attribute argument. Only `meta_attr` is supported.
 --> $DIR/panics_on_unsupported_attr.rs:8:19
  |
8 | #[actix_responder(imaginary_attr = "builder(default)")]
  |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
